src/app/sign-in/components/form-sign-in/form-sign-in.component.ts

```diff
@@ -228,14 +228,20 @@ export class FormSignInComponent implements OnInit, AfterViewInit {
         if (platform.social || platform.institutional) {
           if (this.signInData) {
             const { email, firstName, lastName, providerId } = this.signInData
-            this._router.navigate(['/register'], {
-              queryParams: {
-                ...platform.queryParameters,
-                email,
-                firstName,
-                lastName,
-                providerId,
-              },
+            this._user.getUserSession().subscribe((userSession) => {
```

Comment:
please add the first() pipe, so it close the subscription

Reply:
ohh yeah. I will add it